release-builder.sh

```diff
@@ -1,9 +1,8 @@
-#!/bin/bash -u
+#!/bin/bash 
```

Comment:
Unsure why removed the -u and set -ex yet as haven't read down, but the comment remains

Reply:
I deleted it because an error occurred when executing locally because some GitHub ENV did not exist in local